packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -304,6 +311,30 @@ record whether a new version should be generated when this property is changed.
 This can be typically determined by checking whether the property can be
 modified using the *[UpdateFunctionConfiguration]* API or not.
 
+### `currentVersion`: Updated hashing logic for layer versions
+
+An additional update to the hashing logic fixes two issues surrounding layers.
+Prior to this change, updating the lambda layer version would have no affect on
```

Comment:
yep. version numbers aren't the whole picture because upgrading the version won't get the new behavior. It's the version number when you run `cdk init` that is binding, so I've updated the paragraph to reflect this.